### PR TITLE
[3.5] add mix version test of adding last ver to current ver.

### PR DESCRIPTION
**Pull request by siyuanfoundation**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

Uncomment the test now that 3.4.32 is released.

### CODE REVIEW

File: tests/framework/e2e/etcd_process.go [hunk 1]
```diff
@@ -147,7 +147,11 @@ func (ep *EtcdServerProcess) Start() error {
 		}
 	}
 	ep.cfg.lg.Info("starting server...", zap.String("name", ep.cfg.Name))
-	proc, err := SpawnCmdWithLogger(ep.cfg.lg, append([]string{ep.cfg.ExecPath}, ep.cfg.Args...), ep.cfg.EnvVars)
+	args := ep.cfg.Args
+	if ep.cfg.ExecPath == BinPathLastRelease {
+		args = append(args, "--next-cluster-version-compatible")
```
Comment (serathius) [inline tests/framework/e2e/etcd_process.go:152]: Please make it a config option, and provide it in the test.

**Comment by siyuanfoundation** [comment]
cc @ahrtr @serathius

**Review by serathius — APPROVED** [review]

**Pull request merged**